dev/io.openliberty.org.eclipse.jdt.core.java17/bnd.bnd

```diff
@@ -0,0 +1,24 @@
+#*******************************************************************************
+# Copyright (c) 2023 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Bundle-Name: JDT Compiler
+Bundle-SymbolicName: io.openliberty.org.eclipse.jdt.core.java17
+Bundle-Description: Eclipse Java Compiler (ECJ) from the Java Development Tools (JDT) Project. Version 3.34.0: June, 2023
+
+Import-Package: !*
+   
+Export-Package: org.eclipse.jdt.*;version=3.34.0;usage=JSP
```

Comment:
In the java11 bundle you list the version as `version=3.33.0` we should be consistent with three digits here as well?

Reply:
Not sure what you mean. It's listed as `version=3.34.0;` here.

Reply:
It is! Eyes playing games I guess!